sim/firesim-lib/src/main/cc/fesvr/firesim_tsi.cc

```diff
@@ -1,8 +1,12 @@
 // See LICENSE for license details
+#include <stdio.h>
+#include <inttypes.h>
 #include "firesim_tsi.h"
 
+#define fprintf(stdout, fmt, ...) (0)
```

Comment:
Are you planning on leaving this debug feature in?

Reply:
Yes I think not having the prints print is nice (with users having the option to reenable).